### PR TITLE
record and show the analysis flavor

### LINKED ISSUE #1711: show current flavor (static or dynamic) in default, json, verbose, and vverbose output headers
see title.

### PR DESCRIPTION
**Pull request by williballenthin**
closes #1711 
please review and merge #1709 first

### Checklist

- [x] No CHANGELOG update needed

- [x] No new tests needed

- [x] No documentation update needed

image

### CODE REVIEW

File: capa/render/proto/capa_pb2.py
Comment (williballenthin) [inline capa/render/proto/capa_pb2.py:1]: i used the protobuf-compiler available on debian (versus nix or something else before) and it generates this python-native representation, which i like better than the opaque binary descriptors from before.

Comment (williballenthin) [reply capa/render/proto/capa_pb2.py:1]: kinda explodes the line-count, though

**Review by mr-tz — COMMENTED** [review]
LGTM, but I only glimpsed through, recommend to wait for another review

**Comment by yelhamer** [comment]
LGTM. thanks @williballenthin !

**Review by yelhamer — APPROVED** [review]

**Pull request merged**
